client/scss/core.scss

```
@@ -9,36 +9,25 @@ REFERENCE
   * Classes should be written using BEM, which we modify to include a prefix.
   * Every BEM block should have its own file.
 
-┌────────────┬────────────────────────┬──────────────┬────────┐
-│ Folder     │ Contents               │ ITCSS level  │ Prefix │
-├────────────┼────────────────────────┼──────────────┼────────┤
-│ settings   │ variables, maps, fonts │ 1 Settings   │        │
-│ tools      │ mixins, functions      │ 2 Tools      │        │
-│ generic    │ resets                 │ 3 Generic    │        │
-│ elements   │ elements, no classes   │ 4 Elements   │        │
-│ objects    │ classes                │ 5 Objects    │ o-     │
-│            │   layout               │              │        │
-│ components │ classes                │ 6 Components │ c-     │
-│            │   BEM blocks           │              │        │
-│ overrides  │ classes                │ 7 Trumps     │ u-     │
-│            │   overrides, utilities │              │        │
-└────────────┴────────────────────────┴──────────────┴────────┘
+┌────────────┬────────────────────────┬───────────────┬
+│ Folder     │ Contents               │ ITCSS level  │
+├────────────┼────────────────────────┼───────────────┼
+│ settings   │ variables, maps, fonts │ 1 Settings   │
+│ tools      │ mixins, functions      │ 2 Tools      │
+│ generic    │ resets                 │ 3 Generic    │
+│ elements   │ elements, no classes   │ 4 Elements   │
+│ objects    │ Unused                 │ 5 Objects    │
+│ components │ classes                │ 6 Components │
+│            │   BEM blocks           │              │
+│ overrides  │ classes                │ 7 Trumps     │
+│            │   overrides, utilities │              │
+└────────────┴────────────────────────┴───────────────┘
```

The `|` is no longer aligned

```suggestion
┌────────────┬────────────────────────┬──────────────┬
│ Folder     │ Contents               │ ITCSS level  │
├────────────┼────────────────────────┼──────────────┼
│ settings   │ variables, maps, fonts │ 1 Settings   │
│ tools      │ mixins, functions      │ 2 Tools      │
│ generic    │ resets                 │ 3 Generic    │
│ elements   │ elements, no classes   │ 4 Elements   │
│ objects    │ Unused                 │ 5 Objects    │
│ components │ classes                │ 6 Components │
│            │   BEM blocks           │              │
│ overrides  │ classes                │ 7 Trumps     │
│            │   overrides, utilities │              │
└────────────┴────────────────────────┴──────────────┘
```